packages/fuels/test/features/deploy.test.ts

```diff
@@ -199,6 +203,9 @@ describe('deploy', { timeout: 180000 }, () => {
         wallet
       );
 
+      const { value: getCountValue } = await targetContract.functions.get_value().get();
+      expect(getCountValue).toBe(10);
```

Comment:
I should not have duplicated this `executeTargetContract` method.

If we make it receive a contract ID as a parameter, it can be moved outside the test cases and be used by both.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3235/commits/adf9f15d1460e31677cfacec3e5c075a1478d236